pages/content/amp-dev/documentation/guides-and-tutorials/start/bento_guide.md

```diff
@@ -0,0 +1,343 @@
+---
+$title: Use AMP components in non-AMP pages
+$order: 1
+description: 'Use a subset of AMP components as standalone elements in non-AMP pages with Bento AMP.'
+tutorial: false
+toc: true
+formats:
+  - websites
+author: CrystalOnScript
+---
+
+Use AMP's well-tested, cross browser compatible and accessible components everywhere with Bento AMP. Bento AMP allows you to use AMP components in non-AMP pages without needing to commit to fully valid AMP! Bento allows you to take components and place them in implementations with frameworks and CMSs that don't support AMP. 
+
+Bento AMP enables you to insert highly performant and tested web components, such as carousels and media embeds, anywhere you need. Developers can use Bento components for one-off cases, such as adding a carousel to an otherwise non-AMP page. Bento is also a way of testing out AMP components to see if the path to valid AMP is right for your use case.
+
+[tip type="important"]
+ Bento is only available [experimentally](../learn/experimental.md). Read our [Blog Post](TODO) for information on future plans and how to get involved! 
+[/tip]
+
+# Use Bento components in non-AMP pages
+
+Bento components are available for experimental use in the developer preview. This means you must enable the experimental flag either in the document or the console. This is no different from other [experiments in AMP](../learn/experimental.md). 
+
+The risk of using the Bento experiment is also the same. The AMP teams reserves the right to change the API drastically based on feedback. Changes are necessary to improve features.
```

Comment:
I wouldn't talk about "the risk" or "drastically". I think this paragraph should be replaced by a sentence that just says that, while Bento's in developer preview, APIs may change. Otherwise it kind of sounds like we could keep changing the API indefinitely.

Reply:
Good call - thanks!